s/aspects/utilities/unix-normalize.ts

```diff
@@ -0,0 +1,33 @@
+import { getGenerator } from "../get-generator.js";
+
+function unixNormalize(path: string) {
```

Comment:
here's the ticket! this algorithm is the bugfix.

i'll make some adjustments
- [ ] fix indentation to tabs
- [ ] use inline `export function` statement